src/utils/transaction-guards.ts

```diff
@@ -124,8 +125,12 @@ export const isSignableBy = (txSummary: TransactionSummary, walletAddress: strin
   return !!executionInfo?.missingSigners?.some((address) => address.value === walletAddress)
 }
 
-export const isExecutable = (txSummary: TransactionSummary, walletAddress: string): boolean => {
-  if (!txSummary.executionInfo || !isMultisigExecutionInfo(txSummary.executionInfo)) {
+export const isExecutable = (txSummary: TransactionSummary, walletAddress: string, safe: SafeInfo): boolean => {
+  if (
+    !txSummary.executionInfo ||
+    !isMultisigExecutionInfo(txSummary.executionInfo) ||
+    safe.nonce !== txSummary.executionInfo.nonce
```

Comment:
What if it's lower than the safe nonce?

Reply:
We use the same strict comparison in SignOrExecute tho.

Reply:
Maybe it's good to not allow to skip nonces.

Reply:
Just to summarise why we resolved this: you can't set a nonce lower than the Safe nonce and when it's +2> than the Safe nonce, it's not executable.